build

```diff
@@ -1,5 +1,18 @@
 #!/usr/bin/env bash
 
+usage() {
+  echo "Usage: $0 [GOOS] [GOARCH]"
+}
+
+if [[ -z "$1" ]] || [[ -z "$2" ]]; then
+    DEFAULT_GOOS=`go env | grep GOOS`
+    DEFAULT_GOARCH=`go env | grep GOARCH`
```

Comment:
Please resolve ShellCheck CI failures. "Use $(..) instead of legacy `..`. [SC2006]"

Reply:
sure, fix it in next patch